docs/index.rst

```diff
@@ -280,7 +284,7 @@ Its embedded producer is configured to be idempotent, exactly-once in order sema
    .. figure:: images/ksql_properties.png
       :alt: image
 
-#. This example creates two streams ``EN_WIKIPEDIA_GT_1`` and ``EN_WIKIPEDIA_GT_1_COUNTS``, and the reason is to demonstrate how ksqlDB windows work. ``EN_WIKIPEDIA_GT_1`` counts occurences with a tumbling window, and for a given key it writes a `null` into the table on the first seen message.  The underlying Kafka topic for ``EN_WIKIPEDIA_GT_1`` does not filter out those nulls, but to send just the counts greater than one downstream, there is a separate Kafka topic for ``EN_WIKIPEDIA_GT_1_COUNTS`` which does filter out those nulls (e.g., the query has a clause ``where ROWTIME is not null``).  From the bash prompt, view those underlying Kafka topics.
+#. This example creates two streams ``EN_WIKIPEDIA_GT_1`` and ``EN_WIKIPEDIA_GT_1_COUNTS``, and the reason is to demonstrate how ksqlDB windows work. ``EN_WIKIPEDIA_GT_1`` counts occurrences with a tumbling window, and for a given key it writes a `null` into the table on the first seen message.  The underlying Kafka topic for ``EN_WIKIPEDIA_GT_1`` does not filter out those nulls, but to send just the counts greater than one downstream, there is a separate Kafka topic for ``EN_WIKIPEDIA_GT_1_COUNTS`` which does filter out those nulls (e.g., the query has a clause ``where ROWTIME is not null``).  From the bash prompt, view those underlying Kafka topics.
```

Comment:
`EN_WIKIPEDIA_GT_1` is a topic name. `EN_WIKIPEDIA_GT_1_STREAM` is the stream name.

The `where rowtime is not null` condition is somewhat confusing, because the null-valued records actually have a timestamp. It's the fact that there's a `where` clause that causes the null-valued records to be silently dropped: https://github.com/confluentinc/ksql/issues/5634 

By the way, this behavior will likely be changing in ksqlDB 6.1.0: https://github.com/confluentinc/ksql/issues/3558

Reply:
@mikebin 👍 thank you for surfacing this and flagging the potential behavior change.  Since this PR is for troubleshooting steps (and spelling errors), I filed https://confluentinc.atlassian.net/browse/DEVX-2298 to track the issue separately.